pkgs/development/compilers/rust/clippy-wrapper.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  runCommand,
+  clippy-unwrapped,
+  makeWrapper,
+  sysroot,
```

Comment:
Do we maybe want to call this `rustc`, so we don't need to change the name when we call the package?

Reply:
sure